test/functional/p2p_permissions.py

```diff
@@ -39,21 +39,21 @@ def run_test(self):
         self.checkpermission(
             # default permissions (no specific permissions)
             ["-whitelist=127.0.0.1"],
-            ["relay", "noban", "mempool"],
+            ["relay", "noban", "mempool", 'download'],
```

Comment:
If the default `-whitelist` permissions are changed by this PR, consider pulling in the changes from https://github.com/jonatack/bitcoin/commits/whitelist-whitebind-doc-improvements.

nit: follow style for quotes already in use in this test

Reply:
Ah, good point. Added a comment in the test `# Make sure the default values in the command line documentation match the ones here` and updated the help